src/ScrollyVideo.jsx

```diff
@@ -1,44 +1,79 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useEffect, useRef } from 'react';
 import ScrollyVideo from './ScrollyVideo';
 
-function ScrollyVideoComponent(props) {
-
-  // variable to hold the DOM element
+function ScrollyVideoComponent({
+  src,
+  transitionSpeed,
+  frameThreshold,
+  cover,
+  sticky,
+  full,
+  trackScroll,
+  useWebCodecs,
+  videoPercentage,
```

Comment:
I think you forgot the `debug` prop here:
```suggestion
  videoPercentage,
  debug,
```